created the volunteering opportunities header

Dprof-in-tech opened the pull request:
# 🚀 VolunChain Pull Request

Mark with an `x` all the checkboxes that apply (like `[x]`)

- [x] Closes #11
- [ ] Added tests (if necessary)
- [ ] Run tests
- [ ] Run formatting
- [x] Evidence attached
- [x] Commented the code

---

### 📌 Type of Change

- [ ] Documentation (updates to README, docs, or comments)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

---

## 📝 Changes description

Built the Volunteering opportuniities header screen. it is accessible at /opportunities

---

## 📸 Evidence (A photo is required as evidence)

<img width="1512" alt="Screenshot 2025-01-23 at 17 33 06" src="https://github.com/user-attachments/assets/84018adf-3a7a-4b7d-89da-a96fe58c0291" />


---

## ⏰ Time spent breakdown



---

## 🌌 Comments



---

Thank you for contributing to VolunChain! We appreciate your effort and commitment to this project. Together, we can make a meaningful impact and take VolunChain to the next level!

KevinLatino reviewed (changes requested):
Almost done! But it's a bit bigger for computer resolution, pls make all smaller, and also make the search bar smaller, as in the Figma Design:

<img width="742" alt="Captura de pantalla 2025-01-23 a la(s) 11 42 35 p  m" src="https://github.com/user-attachments/assets/87ec8571-1340-4e74-b88c-31b30cb7f861" />

KevinLatino reviewed (changes requested):
Also pls, let's move this to a component, crearte a folder inside src/components/volunteer-opportunities/header.tsx

KevinLatino reviewed (commented):
Finally, resolve merge conflicts

Dprof-in-tech commented:
> Almost done! But it's a bit bigger for computer resolution, pls make all smaller, and also make the search bar smaller, as in the Figma Design:
> 
> <img alt="Captura de pantalla 2025-01-23 a la(s) 11 42 35 p  m" width="742" src="https://private-user-images.githubusercontent.com/111583322/406317550-87ec8571-1340-4e74-b88c-31b30cb7f861.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc3MDE4NTYsIm5iZiI6MTczNzcwMTU1NiwicGF0aCI6Ii8xMTE1ODMzMjIvNDA2MzE3NTUwLTg3ZWM4NTcxLTEzNDAtNGU3NC1iODhjLTMxYjMwY2I3Zjg2MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEyNFQwNjUyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hM2FlZjdlNDhhYmVlNjU2YzU1MDQ5N2IzNzkxMTA0MDk2YzkwZjcyOTA3OWVmOTQ0MjhmMTE4Y2ZiMmI5ZTFhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.aO0JYw0AQ7NBrfvyOChF78ZmFrQFGkAUMOJ1IDOkA9o">



I am using all the exact measurements in the figma design.

the only problem is that the figma screen is designed for a screen width and heigh of the following dimensions 

width: Fixed (1,920px)px;
height: Fixed (3,426px)px;


but wider laptop screens are of width 1440px to 1620px

this falls in the range of monitor screens which is why my design looks bigger than the figma file but they are all using the same exact dimensions.

KevinLatino reviewed (changes requested):
I understand, but we are using figma as a design reference, we don't think about sizes, so I would appreciate if you could make everything smaller, and there is also a problem with the buttons, when I run the version of your code it looks like this:

<img width="1512" alt="Captura de pantalla 2025-01-24 a la(s) 1 07 17 a  m" src="https://github.com/user-attachments/assets/4b81cd97-3747-4d13-afc9-1f96e75e9f2f" />
